automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/FireCloudClient.scala

```diff
@@ -0,0 +1,113 @@
+package org.broadinstitute.dsde.firecloud.api
+
+import akka.actor.ActorSystem
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model.HttpMethods._
+import akka.http.scaladsl.model.headers.{Authorization, OAuth2BearerToken}
+import akka.http.scaladsl.model.{Multipart, _}
+import akka.stream.ActorMaterializer
+import akka.stream.scaladsl.{Sink, _}
+import akka.util.ByteString
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.scala.DefaultScalaModule
+import org.broadinstitute.dsde.firecloud.config.AuthToken
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor, Future}
+
+class FireCloudClient {
```

Comment:
Looking below, `FireCloudClient` is extended by other traits. So it might be better to make `FireCloudClient` a trait itself. (I didn't even realize that a trait could extend a class: https://stackoverflow.com/questions/12854941/why-can-a-scala-trait-extend-a-class).